specs/global/interfaces.json

```diff
@@ -108,7 +108,7 @@
         "type": "object",
         "properties": {
             "snaks": {
-                "$ref": "#/SnakMap"
+                "$ref": "#/Snaks"
```

Comment:
I _think_ these will go away with a rebase vs #93 (in 363e588 & e966fb3 respectively).

Reply:
:+1: Thanks, I didn't notice that. Rebased...